editor/src/clj/editor/scene_visibility.clj

```diff
@@ -336,7 +341,7 @@
 
         [filters-enabled-control filters-enabled-update-fn]
         (make-toggle {:label "Visibility Filters"
-                      :acc (keymap/key-combo->display-text "Shift+Shortcut+I")
```

Comment:
I think it's not good that we have hardcoded shortcuts here. I changed it to make sure that we use the actual keymap and command name for getting a shortcut from the keymap